src/style.css

```diff
@@ -0,0 +1,132 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
- Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.